### PR TITLE
feat: remove open in new tab because with breadcrumbs not needed anymore. closes #356

### LINKED ISSUE #356: Limit or get rid of opening of new tabs
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I cannot find the right tab back in my web browser, because emx2 opens that many tabs.

**Describe the solution you'd like**
Less opening of new tabs

**Describe alternatives you've considered**
The alternative is already present: we can navigate the system via the breadcrumbs and the banner.

**Additional context**

### PR DESCRIPTION
**Pull request by mswertz**

### CODE REVIEW

File: apps/updownload/src/components/Import.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <Molgenis v-model="session">
+    taskList {{ taskList }}
```
Comment (sidohaakma) [inline apps/updownload/src/components/Import.vue:3]: This is another feature I think? Maybe put it in a separate PR or at least in a separate commit? To keep the changelog accurate?

**Review by sidohaakma — CHANGES_REQUESTED** [review]

File: apps/updownload/src/components/Import.vue (hunk 1 above)
Comment (mswertz) [reply apps/updownload/src/components/Import.vue:3]: ah, well spotted. This is an error should not have been committed. (is part of another feature under development)

**Comment by sidohaakma** [comment]
[remove open in new tab because with breadcrumbs not needed anymore](https://trello.com/c/W371jscS/182-remove-open-in-new-tab-because-with-breadcrumbs-not-needed-anymore)

**Review by sidohaakma — APPROVED** [review]
